apps/design/backend/src/store.ts

```diff
@@ -74,6 +76,52 @@ export interface BallotStyle {
   partyId?: PartyId;
 }
 
+export interface BackgroundTask {
+  id: Id;
+  taskName: string;
```

Comment:
Will likely make this a stricter union of string literals once the kinds of background tasks become clearer